linux_os/guide/system/apparmor/all_apparmor_profiles_in_enforce_complain_mode/ansible/shared.yml

```diff
@@ -0,0 +1,27 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_sle
```

Comment:
I believe you can remove: multi_platform_rhel,multi_platform_fedora,multi_platform_ol